website/docs/components/dropdown/partials/guidelines/guidelines.md

```diff
@@ -316,6 +316,23 @@ Avoid inconsistent icon use.
 </div>
 !!!
 
+### Badge usage
+
+[Badges](/components/badge) in Interactive ListItems are optional. We recommend using only the `small` Badge option to align more closely with the text line height. Badges should be used sparingly and only when their use adds contextual value. Examples of appropriate use of Badge inside an Interactive ListItem include:
+
+- When an item is in beta or behind a feature flag, i.e., a new item that users should be able to locate easily.
```

Comment:
@heatherlarsen I feel confident about the use of i.e. vs e.g. in the other two but this bullet I'm unsure of. Your suggestion was i.e. but a new item seems like it wouldn't be the sole use case here so e.g. feels more appropriate unless you feel strongly?

Reply:
Hmm, you might be right. I don't feel strongly about it.

Reply:
Thanks! I will keep this as i.e. then